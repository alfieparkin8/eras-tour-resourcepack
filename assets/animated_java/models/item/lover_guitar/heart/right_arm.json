{"parent":"animated_java:item/lover_guitar/right_arm","textures":{"2":"minecraft:block/outfits/lover/heart_layer_1"}}