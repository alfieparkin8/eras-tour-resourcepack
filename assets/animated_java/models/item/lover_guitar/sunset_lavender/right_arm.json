{
	"parent": "animated_java:item/lover_guitar/right_arm",
	"textures": {
		"2": "animated_java:item/lover_guitar/lover_lavender"
	}
}