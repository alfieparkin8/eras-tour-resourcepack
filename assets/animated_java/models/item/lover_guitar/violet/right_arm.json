{"parent":"animated_java:item/lover_guitar/right_arm","textures":{"2":"minecraft:block/outfits/lover/violet_layer_1"}}