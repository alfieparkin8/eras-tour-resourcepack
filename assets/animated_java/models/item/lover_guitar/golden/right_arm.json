{"parent":"animated_java:item/lover_guitar/right_arm","textures":{"2":"minecraft:block/outfits/lover/golden_layer_1"}}